### PR TITLE
Fixed the issue on mobile where the footer of the map was not visible

### PR DESCRIPTION
**Pull request by Xamyrz**
Fixed the issue on mobile where the footer of the map was not visible untill scrolled down.

### CODE REVIEW

File: resources/js/views/global/GlobalMapContainer.vue [hunk 1]
```diff
@@ -14,11 +19,16 @@ import Supercluster from './Supercluster'
 export default {
     name: 'GlobalMapContainer',
     components: { Loading, Supercluster },
+    data() {return {mapHeight: window.outerHeight - 72 + "px"}},
```
Comment (xlcrr) [inline resources/js/views/global/GlobalMapContainer.vue:22]: Can you open this line please 

```javascript
data () {
    return {
        mapHeight: window.outerHeight - 72 + "px";
    };
}
```

File: resources/js/views/global/GlobalMapContainer.vue [hunk 2]
```diff
@@ -27,13 +37,29 @@ export default {
         {
             return this.$store.state.globalmap.loading;
         }
+    },
+    methods: {
+        resizeHandler () {
+            console.log(window.innerHeight);
```
Comment (xlcrr) [inline resources/js/views/global/GlobalMapContainer.vue:43]: remove log

File: resources/js/views/global/GlobalMapContainer.vue [hunk 3]
```diff
@@ -27,13 +37,29 @@ export default {
...
+            this.mapHeight = window.innerHeight - 72 + "px";
+        },
+        isPhone(){
+            if(/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)){
+                window.addEventListener("resize", this.resizeHandler);
+                return true;
+            }
+            return false;
+        }
```
Comment (xlcrr) [inline resources/js/views/global/GlobalMapContainer.vue:52]:
```javascript
isMobile ()
{
     return (Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
}
```

File: resources/js/views/global/GlobalMapContainer.vue [hunk 4]
```diff
@@ -27,13 +37,29 @@ export default {
...
     }
 }
 </script>
 
 <style scoped>
     @import '~leaflet/dist/leaflet.css';
 
+    html, body{
+        height: 100%;
+    }
```
Comment (xlcrr) [inline resources/js/views/global/GlobalMapContainer.vue:62]: These should be on the global `openlittermap.css` file

**Review by xlcrr — CHANGES_REQUESTED** [review]
see comments

File: resources/js/views/global/GlobalMapContainer.vue (hunk 3 above)
Comment (Xamyrz) [reply resources/js/views/global/GlobalMapContainer.vue:52]: the if statement needs to stay there to allow for the event handler when on mobile

**Comment by xlcrr** [comment]
I am convinced a css-only solution would be enough here. Can anyone give a 2nd opinion?

**Pull request merged**